packages/events-api/tsconfig.json

```diff
@@ -0,0 +1,39 @@
+{
+  "compilerOptions": {
+    "target": "es2017",
```

Comment:
For some reason I thought the test would catch this, but we don't test against 4 😳

Running on Node.js v4.9.1, I found out `"es2017"`, `"es2016"`, `"es2015"`, and `"es6"` all fail because v4 doesn't support destructuring (i.e. `constructor(signingSecret, { x = false, y = false, z = false } = {})`) and default parameter values (and possibly more, that's just the first error).

So the right target is `"es5"` which produces 0 syntax errors in v4.9.1.